app/javascript/components/organisms/Modal.js

```diff
@@ -1,37 +1,83 @@
 import React from 'react'
 import Icon from '../atoms/Icon'
+import ReactModal from 'react-modal';
+import _ from 'lodash'
 
-const Modal = ({ header, hidden, content, primary, secondary, icon }) => {
-  return (
-    <div aria-labelledby="modalTitle" aria-hidden={hidden} role="dialog">
-      <header className="modal-inner">
-        <h1 className="modal-title t-gamma no-margin">{header}</h1>
-      </header>
-
-      <section className="modal-inner">
-        <p className="c-steel">{content}</p>
-      </section>
-
-      <footer className="modal-footer bg-dust">
-        <div className="modal-button-group row">
-          <div className="modal-button_item">
-            <button className="button primary">{primary}</button>
-          </div>
-          <div className="modal-button_item">
-            <button className="button no-border">{secondary}</button>
-          </div>
-        </div>
-      </footer>
-
-      <a className="close-reveal-modal" aria-label="Close">
-        <span className="ui-icon ui-medium i-primary">
-          <svg>
-            <use xlinkHref="#i-close"></use>
-          </svg>
-        </span>
-      </a>
+import Alert from '../organisms/Alert'
+
+const styleTypes = {
+  small: {
```

Comment:
@slowbot @fddayan can we migrate this to the PL or on overwrites CSS in LAP so that the styling doesn't live here?

Reply:
We could create custom classes for the modal and overlay and I pass that classes to `react-modal`
or we could extend `ReactModal__Overlay` and `ReactModal__Content`

Reply:
@chrisdolendo Jesse is out for a week. Should we go merge this PR? other stories are blocked.